modules/ztp-setting-managed-bare-metal-host-kernel-parameters-manual-workflow.adoc

```diff
@@ -0,0 +1,74 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp_far_edge/ztp-deploying-far-edge-sites.adoc
+
+[id="setting-managed-bare-metal-host-kernel-parameters_{context}"]
+= Setting managed bare-metal host kernel parameters
+
+Specify kernel parameters for managed bare-metal hosts to customize the host's boot process. This is useful for clusters with specific environmental requirements. For example, configure the `rd.net.timeout.carrier` kernel parameter to facilitate static networking or to receive a DHCP address before downloading the root file system. 
+
+[NOTE]
+====
+In {product-title} {product-version}, you can only add kernel parameters. You can not replace or delete kernel parameters.
+====
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (oc).
+* You have logged in to the hub cluster as a user with cluster-admin privileges.
+* You have manually generated the installation and configuration custom resources (CRs).
+
+.Procedure
+
+. Edit the `spec.kernelArguments` specification in the `InfraEnv` CR to configure kernel parameters:
+
+[source,yaml,options="nowrap",role="white-space-pre"]
+----
+apiVersion: agent-install.openshift.io/v1beta1
+kind: InfraEnv
+metadata:
+  name: <cluster_name>
+  namespace: <cluster_name>
+spec:
+  kernelArguments:
+    - operation: append <1>
+      value: audit=0 <2>
+    - operation: append
+      value: trace=1
+  clusterRef:
+    name: <cluster_name>
+    namespace: <cluster_name>
+  pullSecretRef:
+    name: pull-secret
+----
+<1> Specify the append operation to add a kernel argument.
+<2> Specify the kernel argument you want to configure. This example configures the audit kernel parameter and the trace kernel parameter.
+
+[NOTE]
+====
+The `SiteConfig` CR generates the `InfraEnv` resource as part of the day-0 installation CRs.
```

Comment:
Is SiteConfig relevant here? Isn't this flow without gitops ?

Reply:
If you look at step 5 in "Generating ZTP installation and configuration CRs manually" this says you need to process SiteConfig to generate the installation CRs.
![image](https://user-images.githubusercontent.com/104497497/215741347-dc0f0b0a-63aa-4e7f-b5ee-6644cfcc2a66.png)

Reply:
So I think this is OK?